### PR TITLE
Add triangle test

### PR DESCRIPTION
**Pull request by NissimHadar**
# Description
Adds delays to a number of tests
Removes the procedural texture test (for now)
Corrects images due to the fix in model resizing.
Improves sensitivity of reticle test (larger cursor and flat background)

### CODE REVIEW

File: tests/engine/render/material/albedo/test.js [hunk 1]
```diff
@@ -24,6 +24,13 @@ autoTester.perform("Effects of albedo on various materials", Script.resolvePath(
         createdEntities = addCases(TEST_CASES, true, true, autoTester.getOriginFrame());
         validationCamera_translate(OFFSET);
     });
+
+    var fxaaWasOn;
+    autoTester.addStep("Turn off TAA for this test", function () {
+        fxaaWasOn = Render.getConfig("RenderMainView.Antialiasing").fxaaOnOff;
+        Render.getConfig("RenderMainView.JitterCam").none();
+        Render.getConfig("RenderMainView.Antialiasing").fxaaOnOff = true;
+    });
```
Comment (SamGondelman) [inline tests/engine/render/material/albedo/test.js]: Seems like we should add this to the autotester somehow so we don't have to copy it everywhere?

Comment (NissimHadar) [reply tests/engine/render/material/albedo/test.js]: I was thinking the same.  I was hesitant to do this at the beginning, but it seems that TAA should be disabled for all tests (bar one...)
I will do that.

Comment (NissimHadar) [reply tests/engine/render/material/albedo/test.js]: Tests have been modified - TAA disabling now part of autoTester.js

File: tests/utils/autoTester.js [hunk 2]
```diff
@@ -257,6 +260,11 @@ setUpTest = function(testCase) {
     if (!isManualMode()) {
         Menu.setIsOptionChecked("Desktop", true);
     }
+
+    // Disable TAA
```
Comment (samcake) [inline tests/utils/autoTester.js]: I m opposed to that
can we run with TAA on by default instead of off by default

Comment (NissimHadar) [reply tests/utils/autoTester.js]: The problem is that this causes many tests to fail; and I cannot raise the threshold, making the tests less sensitive.

**Pull request merged**